src/theme/material/form.m.css

```diff
@@ -30,4 +31,5 @@
 .fieldRoot {
 	flex: 1;
 	padding: 0 var(--spacing);
+	min-width: calc(20 * var(--spacing));
```

Comment:
Is 16px * 20 the right min size? This collapses the three text input row to two rows even at full width for me.

Reply:
Ah, good catch. Updated.